docs/tests.md

```diff
@@ -10,8 +10,15 @@ Here are the most important options. Fore more details, please use `poe --help`.
   - Run only pyright: `poe pyright`
   - Run only pytest: `poe pytest`
   - Run only pre-commit: `poe style`
+  - Run mypy and generate a report of the code coverage from the tests: `poe mypy_coverage`
```

Comment:
`poe mypy --coverage`. Could also remove it as we point to `poe --help` for more options.

Reply:
Probably better to remove this line and just have the optional block later.